features/works/work_browse.feature

```diff
@@ -118,3 +118,39 @@ chapter when the chapters are reordered.
   When I browse the "Canonical Fandom" works
     And I follow the recent chapter link for the work "My WIP"
   Then I should be on the 2nd chapter of the work "My WIP"
+
+  Scenario: Kudos link from from work browsing leads to full work page
+  Given I am logged in as "cool_author"
+    And I post the chaptered work "Awesome Work"
```

Comment:
> I might be in a habit of sometimes creating a little bit unrelated test setups to ensure some input variety/worst-case scenario.

Instead of having one scenario that implicitly tests many things in its setup, I think we should have a separate scenario for each thing we want tested. Each test scenario should make it clear what is being tested and use the simplest setup / the fewest steps possible.

In this case though, I agree we don't need to check when the work has a draft chapter.